### DISCUSSION TITLE
Business Process Catalog

### DISCUSSION THREAD

**Discussion by JohanssonStefan** [microsoft/dynamics365patternspractices#14] [Q&A] [answered]
Is there a version of the "Business Process Catalog APR 2023.xlsx" that is ready to import to DevOps as a Epic, feature, requirement structure?

**Comment by edupont04** [comment, marked as answer]
Thank you for your question! If you think the catalog can help you as it looks today, then by all means go ahead. 

I've been told that a team inside Microsoft is working on templates to build projects for business solutions in Azure DevOps, but I don't have a solid timeline on when that team will be able to provide such templates but hopefully later this calendar year.
